pkg/api/ui_handler.go

```diff
@@ -61,8 +61,29 @@ func NoCacheHandler(handler http.Handler) http.Handler {
 	})
 }
 
+// BaseURLHandler ensures that routes under BaseURL (i.e. /api/v1) are returned
+// with ErrInvalidAPIEndpoint.
+func BaseURLHandler(handler http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		fmt.Print("printing the url.path now", r.URL.Path)
```

Comment:
remove Print from final code

Reply:
Ahh yes, thank you for catching this, completely forgot/didn't see to remove this leftover debug print ha

Reply:
Done